src/features/landing/components/AuthSection/AuthForm/constants.ts

```diff
@@ -0,0 +1,73 @@
+import { faker } from '@faker-js/faker';
+
+import { Mock } from './types';
+
+export const fullNamePlaceholder: RegExp = /Mykhailo Svitskyi/;
+export const emailPlaceholder: string = 'vilnaCRM@gmail.com';
+export const passwordPlaceholder: string = 'Create a password';
+
+export const submitButtonText: string = 'Sign-Up';
+export const formTitleText: string = 'Or register on the website:';
+export const nameInputText: string = 'Your name and surname';
+export const emailInputText: string = 'Email';
+export const passwordInputText: string = 'Password';
+export const requiredText: string = 'This field is required';
+export const passwordTipAltText: string = 'Password tip mark';
+
+export const authFormSelector: string = '.MuiBox-root';
+
+export const statusRole: string = 'status';
+export const checkboxRole: string = 'checkbox';
+export const alertRole: string = 'alert';
+export const buttonRole: string = 'button';
+
+export const emptyValue: string = '';
+
+export const fullName: string = faker.person.fullName();
+export const email: string = faker.internet.email();
+export const password: string = faker.internet.password({
+  length: 16,
+  prefix: 'Q9',
+});
+export const randomClientId: string = faker.string.uuid();
+
+export const borderStyle: string = 'border: 1px solid #DC3939';
```

Comment:
i can't understand what we need in this line of code, looks very strange, need to add a more understandable name

Reply:
Fixed